lms/djangoapps/courseware/models.py

```diff
@@ -189,7 +188,6 @@ class XModuleStudentInfoField(XBlockFieldBase):
     """
     Stores data set in the Scope.preferences scope by an xmodule field
     """
-
     class Meta(object):  # pylint: disable=missing-docstring
         unique_together = (('student', 'field_name'),)
 
```

Comment:
This whitespace removal/addition (without a good reason) still needs to be dealt with.